src/screens/Settings.js

```diff
@@ -68,7 +67,6 @@ export class Settings extends React.Component {
       width: 100,
       marginTop: 16,
       marginBottom: 16,
-      backgroundColor: baseStyle.title.backgroundColor,
```

Comment:
Why did you remove this?

Reply:
Same reason as the `HathorHeader` comment above.